javalanglib/src/main/scala/java/lang/Long.scala

```diff
@@ -232,4 +233,22 @@ object Long {
     else if (mp != 0) Integer.toOctalString(mp) + padOctal10(lp)
     else Integer.toOctalString(lp)
   }
+
+  /**
+   * Holds the maximal number of digits for each radix, so that a number of that radix
+   * can be converted to an Integer
+   */
+  private def maxExpForRadix(radix: Int): Int = {
+    @inline
+    @tailrec
+    def loop(exp: Int, m: Long): Int = {
+      val l: Long = m * radix
+      if (l <= Int.MaxValue) loop(exp+ 1, l)
+      else exp
+    }
+
+    if (radix < 2) -1
+    else if (radix == 2) 30
+    else loop(0, 1)
+  }
```

Comment:
I think the best is to delete the code, hard-code the values, and ScalaDoc-comment what these values are. It doesn't matter how they were computed in the first place.